code/modules/events/wisdomcow.dm

```diff
@@ -0,0 +1,16 @@
+/datum/round_event_control/wisdomcow
+	name = "Wisdom cow"
+	typepath = /datum/round_event/wisdomcow
+	max_occurrences = 1
+	weight = 20
```

Comment:
That's as common as an anomaly, are you sure about that?

Reply:
its as common as a lot of very low impact events